### PR TITLE
✨ Add Login page

### LINKED ISSUE #1: Login
As a user I want a quick way to login into the app using my email address and password.

----
- [ ] Add form with email and password input
- [ ] login submit button
- [ ] Add forgot password functionality

### PR DESCRIPTION
**Pull request by lmachens**
fixes #1

The background position is not correct. I will fix it soon 🙏.

This pull request requires an update after #7 is merged.

### CODE REVIEW

File: src/app/components/Icons/AppIcon.tsx [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+  return (
+    <svg
+      xmlns="http://www.w3.org/2000/svg"
+      width="112"
+      height="123.372"
+      viewBox="0 0 112 123.372"
+    >
+      <defs>
+        <linearGradient
+          id="linear-gradient"
```
Comment (jujuzitroni) [inline src/app/components/Icons/AppIcon.tsx]: is this needed?

**Review by dhnazarenko — COMMENTED** [review]
lgtm 🚀

**Review by jujuzitroni — APPROVED** [review]
lgtm!

**Review by Dev-dfm — COMMENTED** [review]
Great work! 🚀

File: src/app/pages/Login/Login.module.css [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+  flex-direction: column;
+  row-gap: 2em;
+  justify-content: center;
+  align-items: center;
+}
+
+.main {
+  z-index: 1;
+  width: 100%;
+  max-width: 500px;
```
Comment (MaxMuecket) [inline src/app/pages/Login/Login.module.css:18]:
```suggestion
  max-width: 31.25em;
```

Comment (J-Bannach) [inline src/app/pages/Login/Login.module.css:18]: Is px a good unit here?

File: src/app/pages/Login/Login.module.css [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+  row-gap: 1em;
+}
+
+.hint {
+  color: var(--hint-color);
+}
+
+.fadeOverlay {
+  position: fixed;
+  top: calc(50% - 76px);
```
Comment (MaxMuecket) [inline src/app/pages/Login/Login.module.css:33]:
```suggestion
  top: calc(50% - 4.75em);
```

**Review by J-Bannach — CHANGES_REQUESTED** [review]

**Review by MaxMuecket — CHANGES_REQUESTED** [review]
Looks good to us. Just a few changes🧸

**Review by SGierszewski — APPROVED** [review]
Good job!! 🥳

**Pull request merged**